app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -1,51 +1,132 @@
+// TODO
+// 1. Read Collection - GET /tasks
+// 2. Read Item - GET /tasks/{id}
+// 3. Read Create - POST /tasks
+// 4. Read Update - PUT/PATCH /tasks/{id}
+// 5. Read Delete - DELETE /tasks/{id}
+
 package com.codesoom.assignment.controllers;
 
-import com.codesoom.assignment.application.TaskService;
 import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.services.TaskService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
+/**
+ * 할 일(Task) 관련 Request를 수신하고 Response를 반환합니다.
+ */
```

Comment:
책임을 생각하세요. 다음은 대안 중 하나입니다. 더 좋게 작성하실 수도 있을 거에요.
```suggestion
/**
 * 할 일과 관련된 http 요청 처리를 담당합니다.
 */
```